docs/source/contributing/index.adoc

```diff
@@ -5,6 +5,7 @@ include::variables.adoc[]
 
 Join the {project} community! Learn about the link:https://github.com/minishift/minishift/blob/master/README.adoc[project] and how to link:https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc[contribute].
 
+- link:https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc#contribution-guidelines[Contribution guidelines]
```

Comment:
only `Guidelines` ? as the page title is `Contributing to Minishift`.

Reply:
@budhrg Not sure what you mean. The PR is for making the contribution guideline visible to users.

Reply:
Works for me as well.